docs/data-modeling/field-types.md

```diff
@@ -1,152 +1,189 @@
+### Semantic types for numeric fields
 
-## Profile
+- Quantity
+- Score
+- Percentage
+- Financial
+  - Currency
+  - Discount
+  - Income
+- Location
+  - Latitude
+  - Longitude
 
-Fields that deal with people.
+### Semantic types for temporal fields
 
-- **Birthday**. Date field.
-- **Company**: Text field.
-- **Email**: Displays as a [mailto](https://en.wikipedia.org/wiki/Mailto) link.
-- **Owner**: Text field.
-- **Subscription**: Text field.
-- **User**: Text field.
+- Creation date
+- Creation time
+- Creation timestamp
+- Joined date
+- Joined time
+- Joined timestamp
+- Birthday
 
-## Date and Time
+### Semantic types for text fields
 
-Date and time field types tell Metabase the field contains datetime values, so it can use date pickers and display time series (e.g., a line chart).
+- Entity name
+- Email
 
-If your database stores datetimes as a number or string, you can [cast that column to a datetime](./metadata-editing.md#casting-to-a-specific-data-type).
+  - URL
+  - Image URL
+  - Avatar URL
 
-- **Cancelation date**
-- **Cancelation time**
-- **Cancelation timestamp**
-- **Creation date**
-- **Creation time**
-- **Creation timestamp**
-- **Deletion date**
-- **Deletion time**
-- **Deletion timestamp**
-- **Updated date**
-- **Updated time**
-- **Updated timestamp**
-- **Join date**
-- **Join time**
-- **Join timestamp**
-- **UNIX Timestamp (Milliseconds)**
-- **UNIX Timestamp (Seconds)**
+- Category
+- Name
+- Title
+- Product
+- Source
+- Location
+  - City
+  - State
+  - Country
+  - ZipCode
 
-See [Date formatting options](../questions/visualizations/table.md#date-formatting-options).
+### Semantic types for collection fields
 
-## Categorical
+- Field containing JSON.
 
-- **Enum**: An abbreviation for “enumerated type,” the value of an enum draws on a predefined list of options. An example of an enum would be a field for the months of the year. This list of twelve options is defined in the makeup of the column, and no options outside this list would be valid.
-- **Product**
-- **Source**: For example, the source of a visitor to your website (such as a search engine or other website).
+  See [Working with JSON](./json-unfolding.md).
 
-## URLs
+## Editing data and semantic types
 
-Metabase can display fields with URLs as images or links.
+Admins and people with [Manage table metadata permissions](../permissions/data.md#manage-table-metadata-permissions) can cast data types and edit semantic types in Admin Table Metadata settings.
 
-- **Avatar Image URL**: Displays the field as an image in table and detail views.
-- **Image URL**: Displays the field as an image in table and detail views.
-- **URL**: Displays the field as a link.
+### Cast data types
 
-## Other
+Data types can't be edited in Metabase directly, but certain data types can be [cast to different types types](./metadata-editing.md#casting-to-a-specific-data-type) to be read differently, like interpreting a numerical data type as a date format.
```

Comment:
```suggestion
Data types can't be edited in Metabase directly, but you can cast certain [data types to different types](./metadata-editing.md#casting-to-a-specific-data-type) so that, for example, Metabase will interpret a text data type as a date type.
```